datacube/drivers/postgis/_fields.py

```diff
@@ -501,6 +502,7 @@ def __init__(
     def value_to_alchemy(self, value):
         raise NotImplementedError("range type")
 
+    @override
```

Comment:
I've been using the [basedpyright LSP](https://docs.basedpyright.com/latest/) and it complains about this with the following error:

```
 Untyped function decorator obscures type of function; ignoring decorator
(reportUntypedFunctionDecorator)
Argument of type "property" cannot be assigned to parameter "method" of type "_F@override" in function "override"
Type "property" is not assignable to type "(...) -> Any"
Type "property" is not assignable to type "(...) -> Any"
                                                                   (reportArgumentType)
```

It's happy if the decorators are around the other way:

```
@property
@override
def ...
```

I'm not sure whether it's a legitimate issue or not, but it does make my editor look tidier...

Reply:
When I added the override decorators I did them consistently, and then immediately after the PR was merged I noticed that PyCharm complained.

It seems Pyright is right (as usual): https://peps.python.org/pep-0698/#limitations-of-setting-override

I've flipped the order for the ones in this PR, do you by any chance have a mass-action in your IDE to automatically fix the previous ones that are wrong?